<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPairItem</class>
 <widget class="QWidget" name="ColorPairItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPairItem</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="colorPairId">
       <property name="text">
        <string>•</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="colorPairName">
       <property name="maxLength">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SOURCE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sourceColor">
       <property name="toolTip">
        <string>Sélectionner une nouvelle couleur</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourceColorHex">
       <property name="inputMask">
        <string>\#&gt;HHHHHHHH</string>
       </property>
       <property name="text">
        <string>#FFFFFFFF</string>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TARGET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="targetColor">
       <property name="toolTip">
        <string>Sélectionner une nouvelle couleur</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="targetColorHex">
       <property name="inputMask">
        <string>\#&gt;HHHHHHHH</string>
       </property>
       <property name="maxLength">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
